src/test/java/org/jabref/logic/importer/util/GrobidServiceTest.java

```diff
@@ -0,0 +1,45 @@
+package org.jabref.logic.importer.util;
+
+import org.jabref.preferences.JabRefPreferences;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class GrobidServiceTest {
+
+    static GrobidService grobidService;
+
+    @BeforeAll
+    public static void setup() {
+        grobidService = new GrobidService(JabRefPreferences.getInstance());
+    }
+
+    @Test
+    public void processValidCitationTest() throws GrobidServiceException {
+        String response = grobidService.processCitation("Derwing, T. M., Rossiter, M. J., & Munro, " +
+                "M. J. (2002). Teaching native speakers to listen to foreign-accented speech. " +
+                "Journal of Multilingual and Multicultural Development, 23(4), 245-259.", 1);
+        assertNotNull(response);
+        assertTrue(response.charAt(0) == '@');
```

Comment:
Please verify that the entry is correctly parsed.

Reply:
59088831fe47aa4e7b25bf75d83473b8626504c3